Collectors/wlcg_converter.py

```diff
@@ -184,21 +184,25 @@ def Convert(source_record):
     return to_return
 
 
-def ConvertGstream(source_record):
+def ConvertGstream(source_record,tpc=False):
 
+    if tpc is True:
```

Comment:
```suggestion
    if tpc:
```

Reply:
you could condense it to an Ternary:
```python
typeT = "tcp" if tcp else "metric"
```

But I don't love ternary.

Reply:
I like the frist one.